doc/installing.rst

```diff
@@ -106,16 +106,16 @@ PypeIt has a few optional dependencies that improve and/or expand functionality.
 
       .. code-block:: console
 
-        pip install "pypeit[pyside2,scikit-image]"
+        pip install "pypeit[pyside,scikit-image]"
```

Comment:
Can we change this to `pyqt` instead?  It emphasizes that `pyqt` should be the default.